### ISSUE TITLE
Mod is literally not working somehow?

### ISSUE THREAD

**Issue by cobaltlol** [nvb-uy/Shield_Overhaul#21]
So idk how to explain this but the mod literally doesn't work, I have marium's soulslike weaponry which had parrying in it so for a while I thought that was from shield overhaul, then i removed soulslike weaponry and couldnt use parrying, slamming etc and even the shield itself doesn't block any hits, I haven't changed anything in shield overhaul configs too. Version is 1.20.1 fabric and mod version is 1.0.3

**Comment by Mysterious-Wrangler** [comment]
Yeah same here. Says its really compatible but it just doesnt work, and i also cant even hear the shield thud and i take damage from blocking.

**Comment by Korsun66** [comment]
same for me

**Comment by Barerock** [comment]
I shield bashed once, but now I can't even block arrows or bash. Arrows go straight through and bashing simply doesn't work.

**Comment by cobaltlol** [comment]
If anyone needs it, I changed my shield overhaul configs to this and it seems to work now
{
  "enable_instant_shield_use": true,
  "blacklisted_shields": [
    "// Items here will not use Shield Overhaul\u0027s mechanics",
    "examplemod:epic_shield"
  ],
  "bosses_immune_to_stun": true,
  "bash_only_on_ground": true,
  "bash_damage": 0.0,
  "bash_cooldown_secs": 2.0,
  "bash_stun_duration_secs": 2.0,
  "add_stun_immunity": true,
  "enable_parrying": false,
  "enable_parry_abuse_prevention": false,
  "parry_abuse_cooldown_secs": 0.5,
  "parry_window_secs": 0.5
}
